deny.toml

```diff
@@ -1,33 +1,34 @@
+[graph]
 all-features = true
 
 [advisories]
-db-path = "~/.cargo/advisory-db"
-db-urls = ["https://github.com/rustsec/advisory-db"]
-vulnerability = "deny"
-unmaintained = "deny"
-yanked = "deny"
-notice = "deny"
+version = 2
 ignore = []
 
 [licenses]
-unlicensed = "deny"
-copyleft = "deny"
-default = "deny"
+version = 2
 allow = [
-  "MIT",
-  "MIT-0",
+  "0BSD",
   "Apache-2.0",
+  "Apache-2.0 WITH LLVM-exception",
+  "BSD-2-Clause",
   "BSD-3-Clause",
+  "BSL-1.0",
+  "CC0-1.0",
   "ISC",
+  "MIT",
+  "MIT-0",
+  "Unlicense",
   "Zlib",
-  "0BSD",
-  "BSD-2-Clause",
-  "CC0-1.0",
 ]
+
 exceptions = [
   { name = "unicode-ident", allow = [
     "Unicode-DFS-2016",
   ] },
+  { name = "symphonia", allow = [
+    "MPL-2.0",
```

Comment:
IMO we may just want to make a blanket exception for weak-copyleft licenses like MPL. Any strong-copyleft (i.e. GPL), should remain disallowed due to their virality.

Reply:
yeah but: https://github.com/bevyengine/bevy/pull/10544#issuecomment-1809371469

Reply:
Yeah, my stance is currently "weak-copyleft only in very optional components, and preference to avoid".

Reply:
OK sounds good to me. If that's the case I think we may just want to only allow it if and only if the crate itself is platform agnostic, which I think Symphonia is (cpal is the platform integration for audio, Symphonia is primarily there to provide encoders/decoders), and to prefer more permissive alternatives where available.